src/Illuminate/Mail/TransportManager.php

```diff
@@ -97,8 +97,8 @@ protected function createMailDriver()
      */
     protected function createMailgunDriver()
     {
-        $client = new HttpClient;
         $config = $this->app['config']->get('services.mailgun', []);
+        $client = new HttpClient(Arr::get($config, 'guzzleOptions', []));
```

Comment:
PHP Fatal error:  Class 'Illuminate\Routing\Arr' not found in /home/travis/build/laravel/framework/src/Illuminate/Routing/Router.php on line 339 on travis